<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>850</width>
    <height>381</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>850</width>
    <height>350</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLineEdit" name="DNI_NIE">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>10</y>
      <width>113</width>
      <height>22</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="DNI_lbl">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>10</y>
      <width>61</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>DNI/NIE</string>
    </property>
   </widget>
   <widget class="QLabel" name="Nombre_lbl">
    <property name="geometry">
     <rect>
      <x>230</x>
      <y>10</y>
      <width>61</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Nombre</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="nombre">
    <property name="geometry">
     <rect>
      <x>290</x>
      <y>10</y>
      <width>151</width>
      <height>22</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="direccion">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>40</y>
      <width>481</width>
      <height>22</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="apellidos">
    <property name="geometry">
     <rect>
      <x>520</x>
      <y>10</y>
      <width>311</width>
      <height>22</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="Apellidos_lbl">
    <property name="geometry">
     <rect>
      <x>450</x>
      <y>10</y>
      <width>61</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Apellidos</string>
    </property>
   </widget>
   <widget class="QLabel" name="direccion_lbl">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>40</y>
      <width>61</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Direncción</string>
    </property>
   </widget>
   <widget class="QLabel" name="CP_label">
    <property name="geometry">
     <rect>
      <x>660</x>
      <y>110</y>
      <width>21</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>CP</string>
    </property>
   </widget>
   <widget class="QLabel" name="Poblacion">
    <property name="geometry">
     <rect>
      <x>360</x>
      <y>110</y>
      <width>61</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Población</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="telefono">
    <property name="geometry">
     <rect>
      <x>670</x>
      <y>40</y>
      <width>161</width>
      <height>22</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="email">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>150</y>
      <width>391</width>
      <height>22</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="telefono_lbl">
    <property name="geometry">
     <rect>
      <x>600</x>
      <y>40</y>
      <width>61</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Teléfono</string>
    </property>
   </widget>
   <widget class="QLabel" name="email_lbl">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>150</y>
      <width>61</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>eMail</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="cp">
    <property name="geometry">
     <rect>
      <x>680</x>
      <y>110</y>
      <width>91</width>
      <height>22</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="poblacion">
    <property name="geometry">
     <rect>
      <x>430</x>
      <y>110</y>
      <width>221</width>
      <height>22</height>
     </rect>
    </property>
   </widget>
   <widget class="QComboBox" name="provincia_cbb">
    <property name="geometry">
     <rect>
      <x>160</x>
      <y>80</y>
      <width>251</width>
      <height>22</height>
     </rect>
    </property>
    <item>
     <property name="text">
      <string>Seleccione una Comunidad Autónoma</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Murcia</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Galicia</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Asturias</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Cantabria</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>País Vasco</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Navarra</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Cataluña</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Comunidad Valenciana</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Andalucía</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Comunidad de Madrid</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Ceuta y Melilla</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Islas Canarias</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>La Rioja</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Islas Baleares</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Extremadura</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Castilla La Mancha</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Castilla León</string>
     </property>
    </item>
   </widget>
   <widget class="QLabel" name="Provincia">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>80</y>
      <width>131</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Comunidad Autónoma</string>
    </property>
   </widget>
   <widget class="QRadioButton" name="rdbtn_hombre">
    <property name="geometry">
     <rect>
      <x>540</x>
      <y>150</y>
      <width>71</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Hombre</string>
    </property>
   </widget>
   <widget class="QRadioButton" name="rdbtn_muje">
    <property name="geometry">
     <rect>
      <x>620</x>
      <y>150</y>
      <width>61</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Mujer</string>
    </property>
   </widget>
   <widget class="QRadioButton" name="rdbtn_NoBinario">
    <property name="geometry">
     <rect>
      <x>690</x>
      <y>150</y>
      <width>81</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>No binario</string>
    </property>
   </widget>
   <widget class="QLabel" name="sex_lbl">
    <property name="geometry">
     <rect>
      <x>500</x>
      <y>150</y>
      <width>31</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Sexo</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="cbx_deudor">
    <property name="geometry">
     <rect>
      <x>630</x>
      <y>180</y>
      <width>71</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Deudor</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="cbx_credito">
    <property name="geometry">
     <rect>
      <x>700</x>
      <y>180</y>
      <width>81</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Crédito</string>
    </property>
   </widget>
   <widget class="QLabel" name="Fnac_lbl">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>180</y>
      <width>121</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Fecha de nacimiento</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_limpiar">
    <property name="geometry">
     <rect>
      <x>300</x>
      <y>300</y>
      <width>93</width>
      <height>28</height>
     </rect>
    </property>
    <property name="text">
     <string>Limpiar</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_aceptar">
    <property name="geometry">
     <rect>
      <x>190</x>
      <y>300</y>
      <width>93</width>
      <height>28</height>
     </rect>
    </property>
    <property name="text">
     <string>Aceptar</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_salir">
    <property name="geometry">
     <rect>
      <x>520</x>
      <y>300</y>
      <width>93</width>
      <height>28</height>
     </rect>
    </property>
    <property name="text">
     <string>Salir</string>
    </property>
   </widget>
   <widget class="QLabel" name="Fnac_lbl_2">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>240</y>
      <width>91</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Observaciones</string>
    </property>
   </widget>
   <widget class="QTextEdit" name="observaciones">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>210</y>
      <width>661</width>
      <height>87</height>
     </rect>
    </property>
   </widget>
   <widget class="QDateEdit" name="nacimiento">
    <property name="geometry">
     <rect>
      <x>160</x>
      <y>180</y>
      <width>101</width>
      <height>22</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="Provincia_2">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>110</y>
      <width>61</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Provincia</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="provincia">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>110</y>
      <width>221</width>
      <height>22</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_borrar">
    <property name="geometry">
     <rect>
      <x>410</x>
      <y>300</y>
      <width>93</width>
      <height>28</height>
     </rect>
    </property>
    <property name="text">
     <string>borrar</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>850</width>
     <height>26</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <tabstops>
  <tabstop>DNI_NIE</tabstop>
  <tabstop>nombre</tabstop>
  <tabstop>apellidos</tabstop>
  <tabstop>direccion</tabstop>
  <tabstop>telefono</tabstop>
  <tabstop>poblacion</tabstop>
  <tabstop>cp</tabstop>
  <tabstop>provincia_cbb</tabstop>
  <tabstop>email</tabstop>
  <tabstop>rdbtn_hombre</tabstop>
  <tabstop>rdbtn_muje</tabstop>
  <tabstop>rdbtn_NoBinario</tabstop>
  <tabstop>cbx_deudor</tabstop>
  <tabstop>cbx_credito</tabstop>
  <tabstop>btn_aceptar</tabstop>
  <tabstop>btn_limpiar</tabstop>
  <tabstop>btn_salir</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
